modules/rostests/apitests/setupapi/SetupDiInstallClassExA.c

```diff
@@ -46,7 +46,7 @@ static void create_inf_file(LPCSTR filename, const char *data)
 static void test_SetupDiInstallClassExA(void)
 {
     char inf[2048];
-    char path[MAX_PATH];
+    char path[MAX_PATH + 20];
```

Comment:
I'd rather prefer strsafe functions over a non-standard buffer size here.
Paths in Win32 should never be longer than MAX_PATH, same for the arrays holding them.